README.md

```diff
@@ -69,17 +65,36 @@ units. One unit represents a payout of one token
 contract is resolved.
 
 The position table uses negative numbers for UNFIXED
-positions, positive numbers for FIXED.  There are no
+positions, positive numbers for FIXED. There are no
 zero positions.
 
-
 ## Getting started
 
+### On Windows
+
+Windows 10. Docker version 20.10.12, build e91ed57. Visual Studio Code v1.65.
```

Comment:
Can you add a link? Is this Docker Desktop for Windows?

Reply:
Alright, Don. I will effect the changes. I did observe a few things from my end, will push it soon.